src/GitHub.App/ViewModels/RepositoryCloneViewModel.cs

```diff
@@ -175,12 +175,8 @@ bool IsAlreadyRepoAtPath(string path)
 
```

Comment:
Let's add a `Debug.Assert(path != null)` here, to protect against accidental regressions of the validator logic that ends up calling this.